src/content/resources/support.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Flutter support
+description: Where can you get support when developing with Flutter.
+---
+
+Welcome!
+
+There are many different ways to get support
+when developing with Flutter.
+As Flutter is an open source project,
+your options range from filing issues when you
+run into bugs in the software,
+to connecting with your local community,
+to browsing and identifying a list of commercial agencies
+and consultants who offer support services.
+
+### I want to report an issue with Flutter
```

Comment:
The checks and build are failing as these headers aren't under a h2. Perhaps they all should be h2 instead?

Reply:
Oh! I didn't know that we were enforcing that.